integration-tests/chopsticks/src/network/hydraDx.ts

```diff
@@ -1,15 +1,22 @@
-import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
-import type { Config } from './types.js'
-import { initialBalanceHDX, initialBalanceKILT, toNumber } from '../utils.js'
+import { SetupOption } from '@acala-network/chopsticks-testing'
 
-/// Options used to create the HydraDx context
-export const options: SetupOption = {
-	endpoint: process.env.HYDRADX_WS || ['wss://hydradx-rpc.dwellir.com', 'wss://rpc.hydradx.cloud'],
-	db: './db/hydradx.db.sqlite',
-	port: toNumber(process.env.HYDRADX_PORT) || 9001,
-}
+import { initialBalanceHDX, initialBalanceKILT, toNumber } from '../helper/utils.js'
 
-export const kiltTokenId = 28
+/// Options used to create the HydraDx context
+export const getSetupOptions = ({
+	blockNumber,
+	wasmOverride,
+}: {
+	blockNumber?: number
+	wasmOverride?: string
+} = {}) =>
+	({
+		endpoint: process.env.HYDRADX_WS || ['wss://hydradx-rpc.dwellir.com', 'wss://rpc.hydradx.cloud'],
+		db: './db/hydradx.db.sqlite',
+		port: toNumber(process.env.HYDRADX_PORT) || 9001,
+		blockNumber,
+		wasmOverride,
+	}) as SetupOption
 
 /// Sets the [TechnicalCommittee] and [Council] governance to the given accounts
 export function setGovernance(addr: string[]) {
```

Comment:
You could group all storage modifiers into a `storage` object, to make it clear these do not do anything other than returing a specific storage component.